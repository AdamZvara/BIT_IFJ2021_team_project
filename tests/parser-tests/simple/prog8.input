require "ifj21"

function foo() : integer, string
	local s : string = "Najdi index, kde se vyskytuje prvni x"
	local index : integer = 1
	local letter : integer = 0
	letter = ord(s,index)
	while letter ~= 120 do
		index = index + 1
		letter = ord(s,index)
		if index >= #s then
			return 0, "NE"
		else end
	end
	return index, "ANO"
end


function main()
	local i : integer = 0
	local s : string = ""
	i,s = foo()
	write("Vyskituje se x v zadanem textu? ",s,"\n")
	if s == "ANO" then
		write("vyskytuje se na indexu ",i,"\n")
	else
		write("NEVYSKYTUJE\n")
	end

end

main()
