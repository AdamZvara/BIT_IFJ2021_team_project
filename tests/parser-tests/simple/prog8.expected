Vyskituje se x v zadanem textu? ANO
vyskytuje se na indexu 11
